spec/brainstem/concerns/controller_dsl_spec.rb

```diff
@@ -691,16 +691,22 @@ module Concerns
                 actions :show do
                   response :hash do |response_param|
                     response_param.fields :mk, :hash do |dk|
-                      dk.dynamic_key_field :string
-                      dk.field :mk_blah, :string, required: true
+                      dk.dynamic_key_field :string,
+                        info: "I am a dynamic key field I"
+                      dk.field :mk_blah, :string,
+                        required: true
                     end
 
                     response_param.fields :mk2, :hash do |dk|
-                      dk.dynamic_key_field :string, required: true
+                      dk.field :_dynamic_key, :string,
```

Comment:
We might want to use the constant `DYNAMIC_KEY`?

Reply:
We wanted to give a clear example of how the name of the field can be used to define a dynamic field